project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.9
```

Comment:
Any reason not to bump this in commons as well?  It might be nice to move our projects to sbt 1.1.x

Reply:
Nevermind, I looked here first.